.gitignore

```diff
@@ -8,4 +8,5 @@ node_modules/
 site/node_modules/
 site/themes/docsy
 internal/pkg/template/templates/custom-resources/
-regression/*/copilot/
\ No newline at end of file
+regression/*/copilot/
```

Comment:
nit: do we need to add coverage.out here as well? 🤷 just asking because you have one for the custom resources

Reply:
it's already there :blush: